cmd/bashbrew/cmd-build.go

```diff
@@ -93,6 +96,15 @@ func cmdBuild(c *cli.Context) error {
 
 					// TODO use "meta.StageNames" to do "docker build --target" so we can tag intermediate stages too for cache (streaming "git archive" directly to "docker build" makes that a little hard to accomplish without re-streaming)
 
+					if fromScratch {
+						// https://github.com/docker/cli/blob/v20.10.7/cli/command/image/build.go#L163
+						os.Setenv("DOCKER_DEFAULT_PLATFORM", ociArch.String())
```

Comment:
I guess this might have some potential side effects since this variable is generic and applies to any command which accepts `--platform`. :disappointed:

We should probably either reset the value after `dockerBuild` or come up with something that sets it only for the `docker build` invocation. :confused: